`Docs`: udpate `azurerm_key_vault_access_policy` object_id description for principal app (#20366)
hashicorp/terraform-provider-azurerm: website/docs/r/key_vault_access_policy.html.markdown

    "Get",
  ]
}

data "azuread_service_principal" "example" {

>   display_name = "expamle-app"

Reviewer: ```suggestion
  display_name = "example-app"
```

}

resource "azurerm_key_vault_access_policy" "example-principal" {

>   key_vault_id = azurerm_key_vault.bug_kv.id

Reviewer: should this be `azurerm_key_vault.example.id`?

* `tenant_id` - (Required) The Azure Active Directory tenant ID that should be used for authenticating requests to the key vault. Changing this forces a new resource to be created.

> * `object_id` - (Required) The object ID of a user, service principal or security group in the Azure Active Directory tenant for the vault. To fetch the object ID of a service principal, please use `azuread_service_principal.object_id` not `azuread_application.object_id`.The object ID must be unique for the list of access policies. Changing this forces a new resource to be created.

Reviewer: ```suggestion
* `object_id` - (Required) The object ID of a user, service principal or security group in the Azure Active Directory tenant for the vault. The object ID of a service principal can be fetched from `azuread_service_principal.object_id`. The object ID must be unique for the list of access policies. Changing this forces a new resource to be created.
```
